src/EA.Iws.Core/Admin/Reports/ComplianceDataGuidance.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace EA.Iws.Core.Admin.Reports
+{
+    using System.ComponentModel;
+
+    public class ComplianceDataGuidance
+    {
+        [DisplayName("Notification number")]
+        public string NotificationNumber { get; set; }
+
+        [DisplayName("Number of shipments with No Prenotification")]
+        public string NoPrenotificationCount { get; set; }
```

Comment:
Had to create a copy of the `ComplianceData` class but updated the `int` properties to `string` to store the guidance text.